# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../packeth/pkg-config.patch
# Copyright (C) 2007 - 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- packETH-1.5/Makefile.vanilla	2007-08-23 11:04:43.000000000 +0200
+++ packETH-1.5/Makefile	2007-08-23 11:08:23.000000000 +0200
@@ -27,12 +27,12 @@
 INSTALL_PREFIX = /usr
 
 CC = gcc
-CPPFLAGS =  -I/usr/include/gtk-2.0 -I/usr/lib/gtk-2.0/include -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include 
+CPPFLAGS = $(shell pkg-config --cflags pango gtk+-2.0 gthread-2.0)
 LDFLAGS = 
 CFLAGS = -g -O2 -Wall -Wunused -Wmissing-prototypes -Wmissing-declarations 
 #LIBS = -L/usr/lib -L/usr/X11R6/lib -lgtk -lgdk -lglib -lXi -lXext -lX11 -lm -lpthread -lgthread
 #LIBS = -lgtk-x11-2.0 -lgdk-x11-2.0 -latk-1.0 -lgdk_pixbuf-2.0 -lm -lpangocairo-1.0 -lpango-1.0 -lcairo -lgobject-2.0 -lgmodule-2.0 -ldl -lglib-2.0 -lpthread -lgthread-2.0
-LIBS = -lgtk-x11-2.0 -lgdk-x11-2.0 -latk-1.0 -lgdk_pixbuf-2.0 -lm -lgobject-2.0 -lglib-2.0 -lpthread -lgthread-2.0
+LIBS = $(shell pkg-config --libs pango gtk+-2.0 gthread-2.0)
 
 SOURCES = \
 	main.c \
